K/KrigingEstimators/Compat.toml

```diff
@@ -9,13 +9,13 @@ julia = "0.6-0"
 ["0.0.2-0.0"]
 Variography = "0.0.2-0"
 
-["0.1-0"]
-Combinatorics = "0.7-0"
-Reexport = "0.2-0"
-
 ["0.1-0.2.1"]
 julia = "0.7-1"
 
+["0.1-0.2.2"]
+Combinatorics = "0.7-0"
+Reexport = "0.2-0"
```

Comment:
Please keep these for 0.2.3

Reply:
I don't need lower bounds which were made in Julia < 1.x